repos/spack_repo/builtin/packages/cufftmp/package.py

```diff
@@ -0,0 +1,54 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import platform
+
+from spack_repo.builtin.build_systems.cuda import CudaPackage
+from spack_repo.builtin.build_systems.generic import Package
+
+from spack.package import *
+
+_versions = {
+    "11.4.0.6": {
+        "Linux-x86_64": dict(
+            sha256="6a0597e10e698ab46ea9a607efe06f681ba6e2bf0d8eddd8a72a09c18a8f7253",
+            url="https://developer.download.nvidia.com/compute/cufftmp/redist/libcufftmp/linux-x86_64/libcufftmp-linux-x86_64-11.4.0.6_cuda12-archive.tar.xz",
+        ),
+        "Linux-aarch64": dict(
+            sha256="c0f944fd6ef2e13ef5adfd48ef2e02660f5f573a69c515580096a9eaae13be16",
+            url="https://developer.download.nvidia.com/compute/cufftmp/redist/libcufftmp/linux-sbsa/libcufftmp-linux-sbsa-11.4.0.6_cuda12-archive.tar.xz",
+        ),
+    }
+}
+
+
+class Cufftmp(Package, CudaPackage):
+    """
+    NVIDIA cuFFTMp library.
+    """
+
+    homepage = "https://docs.nvidia.com/cuda/cufftmp/"
+    url = "https://developer.download.nvidia.com/compute/cufftmp/redist/libcufftmp"
+
+    maintainers("albestro")
+
+    license("UNKNOWN")
```

Comment:
I think something along these lines this looks better:
```suggestion
    # https://docs.nvidia.com/cuda/cufftmp/license.html
    license("Nvidia license for Nvidia math libraries software development kits")
```